src/calculationEngine/entitys/CeEntity.java

```diff
@@ -15,6 +17,7 @@
     private BeastTypes type;
     private Nature nature;
     private CeEntity development;
+    private int developmentLvl;
```

Comment:
Why is it called development and develomentLvl?

Reply:
Because I'm flash at typing